cclib/parser/nwchemparser.py

```diff
@@ -1081,6 +1093,140 @@ def extract(self, inputfile, line):
                     line = next(inputfile)
                 self.atomcoords.append(atomcoords_step)
 
+        # Extract Thermochemistry in au (Hartree)
+        #
+        # have to deal with :
+        # Temperature                      =   298.15K
+        # frequency scaling parameter      =   1.0000
+        # Zero-Point correction to Energy  =  259.352 kcal/mol  (  0.413304 au)
+        # Thermal correction to Energy     =  275.666 kcal/mol  (  0.439302 au)
+        # Thermal correction to Enthalpy   =  276.258 kcal/mol  (  0.440246 au)
+        # Total Entropy                    =  176.764 cal/mol-K
+        # - Translational                =   44.169 cal/mol-K (mol. weight = 448.1245)
+        # - Rotational                   =   37.018 cal/mol-K (symmetry #  =        1)
+        # - Vibrational                  =   95.577 cal/mol-K
+        # Cv (constant volume heat capacity) =  103.675 cal/mol-K
+        # - Translational                  =    2.979 cal/mol-K
+        # - Rotational                     =    2.979 cal/mol-K
+        # - Vibrational                    =   97.716 cal/mol-K
+        if line[1:12] == "Temperature":
+            self.set_attribute('temperature', float(line.split()[2][:-1]))
+        if line[1:28] == "frequency scaling parameter":
+            self.set_attribute('pressure', float(line.split()[4]))
+        if line[1:31] == "Thermal correction to Enthalpy":
+            self.set_attribute('enthalpy', float(line.split()[8]) + utils.convertor(self.scfenergies[-1], "eV", "hartree"))
+        if line[1:32] == "Zero-Point correction to Energy":
+            self.set_attribute('zero_point_energy', float(line.split()[8]) + utils.convertor(self.scfenergies[-1], "eV", "hartree"))
+        if line[1:29] == "Thermal correction to Energy":
+            self.set_attribute('electronic_thermal_energy', float(line.split()[8]) + utils.convertor(self.scfenergies[-1], "eV", "hartree"))
+        if line[1:14] == "Total Entropy":
+            self.set_attribute('entropy', utils.convertor(1e-3 * float(line.split()[3]),"kcal/mol","hartree"))
+        
+        # extract vibrational frequencies (in cm-1)
+        if line.strip() == "Normal Eigenvalue ||           Projected Infra Red Intensities":
+            if not hasattr(self, 'vibfreqs'):
+                self.vibfreqs = []
+            if not hasattr(self, 'vibirs'):
+                self.vibirs = []
+            line = next(inputfile) # units
+            line = next(inputfile) # dashes
+            line = next(inputfile) # first line of data
+            while (line[:-1] != " ----------------------------------------------------------------------------"):
+                self.vibfreqs.append(float(line.split()[1]))
+                self.vibirs.append(float(line.split()[5]))

```

Comment:
here (and everywhere you use `float` to convert) could you use `utils.float()` https://github.com/cclib/cclib/blob/e5f66bd48f3f0a78138e7044753789cb1a403106/cclib/parser/utils.py#L62

Reply:
Can I correct all occurences ? even those that are not mine ?

Reply:
Please do! Much appreciated

Reply:
it should be ok